CIP-0116/cardano-babbage.json

```diff
@@ -0,0 +1,1952 @@
+          "$ref": "cardano-babbage.json#/definitions/BaseAddress"
+        },
+        {
+          "$ref": "cardano-babbage.json#/definitions/PointerAddress"
+        },
+        {
+          "$ref": "cardano-babbage.json#/definitions/EnterpriseAddress"
+        },
+        {
+          "$ref": "cardano-babbage.json#/definitions/ByronAddress"
+        }
+      ]
+    },
+    "Ed25519KeyHash": {
+      "title": "Ed25519KeyHash",
+      "type": "string",
+      "format": "hex",
+      "pattern": "^[0-9a-f]{56}$"
+    },
+    "ScriptHash": {
+      "title": "ScriptHash",
+      "type": "string",
+      "format": "hex",
+      "pattern": "^[0-9a-f]{56}$"
+    },
+    "AssetName": {
+      "title": "AssetName",
+      "type": "string",
+      "format": "hex",
+      "pattern": "^([0-9a-f][0-9a-f]){0,32}$"
+    },
+    "Credential": {
+      "title": "Credential",
+      "type": "object",
+      "discriminator": {
+        "propertyName": "tag"
+      },
+      "oneOf": [
+        {
+          "type": "object",
+          "properties": {
+            "tag": {
+              "enum": [
+                "pubkey_hash"
+              ]
+            },
+            "value": {
+              "$ref": "cardano-babbage.json#/definitions/Ed25519KeyHash"
+            }
+          },
+          "required": [
+            "tag",
+            "value"
+          ],
+          "additionalProperties": false
+        },
+        {
+          "type": "object",
+          "properties": {
+            "tag": {
+              "enum": [
+                "script_hash"
+              ]
+            },
+            "value": {
+              "$ref": "cardano-babbage.json#/definitions/ScriptHash"
+            }
+          },
+          "required": [
+            "tag",
+            "value"
+          ],
+          "additionalProperties": false
+        }
+      ]
+    },
+    "MultiAsset": {
+      "title": "MultiAsset",
+      "description": "A mapping from policy IDs (script hashes) to mappings from asset names to amounts",
+      "type": "object",
+      "patternProperties": {
+        "^[0-9a-f]{56}$": {
+          "type": "object",
+          "patternProperties": {
+            "^([0-9a-f][0-9a-f]){0,32}$": {
+              "$ref": "cardano-babbage.json#/definitions/PosInt64"
+            }
+          },
+          "additionalProperties": false
+        }
+      },
+      "additionalProperties": false
+    },
+    "Value": {
+      "title": "Value",
+      "type": "object",
+      "properties": {
+        "coin": {
+          "$ref": "cardano-babbage.json#/definitions/UInt64"
+        },
+        "assets": {
+          "$ref": "cardano-babbage.json#/definitions/MultiAsset"
+        }
+      },
+      "required": [
+        "coin"
+      ],
+      "additionalProperties": false
+    },
+    "TransactionHash": {
+      "title": "TransactionHash",
+      "type": "string",
+      "format": "hex",
+      "pattern": "^[0-9a-f]{64}$",
+      "maxLength": 64,
+      "minLength": 64
```

Comment:
Why do we specify minLength for transaction hashes but not other things, such as keyHashes or addresses?